hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieTableServiceClient.java

```diff
@@ -549,7 +549,9 @@ private void completeClustering(HoodieReplaceCommitMetadata metadata,
       LOG.info("Committing Clustering {} for table {}", clusteringCommitTime, table.getConfig().getBasePath());
       LOG.debug("Clustering {} finished with result {}", clusteringCommitTime, metadata);
 
-      ClusteringUtils.transitionClusteringOrReplaceInflightToComplete(false, clusteringInstant, metadata, table.getActiveTimeline());
+      HoodieInstant completedClusteringInstant = ClusteringUtils.transitionClusteringOrReplaceInflightToComplete(false, clusteringInstant, metadata, table.getActiveTimeline());
+      // Call Table Format Commit to allow writing supplementary table format.
```

Comment:
nit: remove comment. 

if we do this, the abstractions should ideally naturally flow with the code structure.